src/main/java/pl/piekoszek/gorskimatches/image/EquationController.java

```diff
@@ -0,0 +1,24 @@
+package pl.piekoszek.gorskimatches.image;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+
+@RestController
+@RequestMapping("api/equation/newequation")
+class EquationController {
+
+    private final EquationRandomizer equationRandomizer;
+
+    EquationController(EquationRandomizer equationRandomizer) {
+        this.equationRandomizer = equationRandomizer;
+
+    }
+
+    @GetMapping("randomNumber")
```

Comment:
just `random` (it's equation, not number, and we already have `equation` word in path